lib/dbs/mysql.ts

```diff
@@ -1,10 +1,11 @@
 import * as mysql from 'mysql';
 import { promisify } from 'util';
+import { MYSQL_CONFIG } from '@lib/constants';
 import { SessionProps, StoreData } from '../../types';
 
 // For use with Heroku ClearDB
 // Other mysql: https://www.npmjs.com/package/mysql#establishing-connections
-const connection = mysql.createConnection(process.env.CLEARDB_DATABASE_URL);
+const connection = mysql.createConnection(MYSQL_CONFIG.host ? MYSQL_CONFIG : process.env.CLEARDB_DATABASE_URL);
```

Comment:
it would probably be easier to follow to either build a url here or build the config object in this file

Reply:
we might want to instead check if `CLEARDB_DATABASE_URL` is defined first - since we have a deploy to Heroku button, we might want to give preference to clearDB